components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -191,7 +191,7 @@ public static void createApplication(String appName, String appOwner, String app
         if (CONSOLE_APP.equals(appName) && AppsCommonDataHolder.getInstance().isOrganizationManagementEnabled()) {
             AuthenticationStep authenticationStep1 = new AuthenticationStep();
             LocalAuthenticatorConfig identifierFirst = new LocalAuthenticatorConfig();
-            identifierFirst.setName("IdentifierExecutor");
+            identifierFirst.setName(FrameworkConstants.RequestAttribute.IDENTIFIER_FIRST_AUTHENTICATOR);
```

Comment:
The constant taken from - https://github.com/wso2/carbon-identity-framework/blob/master/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java#L414